addons/mail/static/tests/client_action_tests.js

```diff
@@ -0,0 +1,92 @@
+odoo.define('mail.client_action_test', function (require) {
+"use strict";
+
+var ActionManager = require('web.ActionManager');
+var core = require('web.core');
+var config = require('web.config');
+var testUtils = require('web.test_utils');
+var Widget = require('web.Widget');
+
+var ChatAction = core.action_registry.get('mail.chat.instant_messaging');
```

Comment:
would be better to directly import the dependency: var ChatAction = require('mail.chat_client_action'); 
You will need to add a return ChatAction; at the end of client_action.js.